server/routes.go

```diff
@@ -973,17 +973,15 @@ func Serve(ln net.Listener) error {
 
 	if !envconfig.NoPrune {
 		// clean up unused layers and manifests
-		if err := PruneLayers(); err != nil {
-			return err
-		}
-
-		manifestsPath, err := GetManifestPath()
+		layers, err := Layers()
```

Comment:
This reads like it removes all layers on start-up to me.

Reply:
I updated the method from `Remove` to `Prune` which should should clarify the intent